src/analysis/special_functions/exp_log.lean

```diff
@@ -475,6 +475,20 @@ end
 lemma log_nonpos (hx : 0 ≤ x) (h'x : x ≤ 1) : log x ≤ 0 :=
 (log_nonpos_iff' hx).2 h'x
 
+lemma log_nonzero_of_ne_one (x: ℝ) (hx_pos: 0 < x) (hx: x ≠ 1): real.log x ≠ 0 :=
+begin
+  by_cases (1 < x),
+  exact ne_of_gt (log_pos h),
+  push_neg at h,
+  exact ne_of_lt (log_neg hx_pos (lt_of_le_of_ne h hx)),
+end
+
+lemma log_inj_pos {x y: ℝ} (x_pos: 0 < x) (y_pos: 0 < y):
+  real.log x = real.log y → x = y :=
+λ h, le_antisymm
```

Comment:
Any reason not to move the hypothesis to the left of the colon like this?

```suggestion
lemma log_inj_pos {x y: ℝ} (x_pos: 0 < x) (y_pos: 0 < y)
  (h : real.log x = real.log y) : x = y :=
le_antisymm
```

Reply:
None actually, let's stick to mathlib standards.